src/libs/actions/Wallet.js

```diff
@@ -321,6 +321,36 @@ function answerQuestionsForWallet(answers, idNumber) {
     );
 }
 
+function requestPhysicalExpensifyCard(cardID, {legalFirstName, legalLastName, phoneNumber, address: {city, street, zip}}) {
+    const params = {
+        legalFirstName,
+        legalLastName,
+        phoneNumber,
+        addressCity: city,
```

Comment:
@grgia I followed the specifications but I'm thinking that the country and state is missing here

Reply:
👍  agree, I think we need these. Here are the complete params the API expects

```
authToken,
legalFirstName,
legalLastName,
phoneNumber,
addressCity,
addressStreet,
addressZip,
addressCountry,
addressState
```